run-llvm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+git clone git@github.com:llvm/llvm-project.git
+cd llvm-project
+git checkout remotes/origin/release/15.x
+docker pull openeuler/openeuler:23.03
+docker run -itd --restart=always -v  $PWD:/home/llvm-project --name llvm-chunyu openeuler/openeuler:23.03 /bin/bash
+docker exec llvm-chunyu /bin/sh -c "dnf install -y cmake ninja-build autoconf automake make python3 libmpc-devel mpfr-devel gmp-devel gawk  bison flex texinfo patchutils gcc gcc-c++ zlib-devel expat-devel diffutils;
+cd /home/llvm-project && mkdir rvoe23_03_llvm && cd rvoe23_03_llvm;
+cmake -DLLVM_ENABLE_PROJECTS="clang" -DCMAKE_BUILD_TYPE="Release"  -G Ninja ../llvm;
+ninja;
+cd ../;
+tar -czvf rvoe23_03_llvm.tar.gz rvoe23_03_llvm/bin/;"
```

Comment:
```suggestion
docker exec llvm-chunyu /bin/sh -c "dnf install -y cmake ninja-build autoconf automake make python3 libmpc-devel mpfr-devel gmp-devel gawk  bison flex texinfo patchutils gcc gcc-c++ zlib-devel expat-devel diffutils &&
cd /home/llvm-project && mkdir rvoe23_03_llvm && cd rvoe23_03_llvm &&
cmake -DLLVM_ENABLE_PROJECTS="clang" -DCMAKE_BUILD_TYPE="Release"  -G Ninja ../llvm &&
ninja &&
cd ../ &&
tar -czvf rvoe23_03_llvm.tar.gz rvoe23_03_llvm/bin/;"
```

如果前面出错了就不执行后面的工作了
这个sh本身也改成这样会更好